README.md

```diff
@@ -31,12 +38,12 @@ Add the following to your POM file:
          <!-- These are the default values -->
           <!-- Where to output the generated tests -->
            <outputDirectory>${project.build.directory}/generated-test-sources/cucumber</outputDirectory>
-           <!-- The diectory containing your feature files.  -->
+           <!-- The diectory, which must be in the root of the runtime classpath, containing your feature files.  -->
           <featuresDirectory>src/test/resources/features/</featuresDirectory>
            <!-- Directory where the cucumber report files shall be written  -->
           <cucumberOutputDir>target/cucumber-parallel</cucumberOutputDir>
-          <!-- comma separated list of output formats -->
-         <format>json</format>
+          <!-- comma separated list of output formats. For JUnit Rerun json,html,rerun are mandatory-->
+         <format>json,html,rerun</format>
```

Comment:
ok, but again, rather than requiring the user to know that they need to add "json" and "rerun" to the format, these should be added automatically when rerun configuration is used.